src/qmapshack/gis/CGisWorkspace.h

```diff
@@ -348,9 +349,10 @@ class CGisWorkspace : public QWidget, private Ui::IGisWorkspace
 
     /**
        @brief Add a new waypoint by Position
-       @param pt    the position in [?]
+       @param pt    the position in degrees
      */
     void addWptByPos(QPointF pt, const QString& name = QString::Null(), const QString& desc = QString::Null()) const;
+    void addWptByPos(QList<poi_t> pois, IGisProject *project = nullptr) const;
```

Comment:
const references. For both.

Reply:
`const poi_t&` would cause some serious headache and I doubt it will actually work even if using `std::reference_wrapper`

Reply:
??? But you changed it already. But not in for `QPointF pt`.

Reply:
Ah, okay there was a misuderstanding. I thought you wanted me to change `QList<poi_t> pois` to `const QList<const poi_t&>& pois`